test/TestExpectations.json

```diff
@@ -244,18 +244,32 @@
     "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
   },
   {
-    "testIdPattern": "[cookies.spec] Cookie specs Page.deleteCookie should delete cookie for specified URL regardless of the current page",
+    "testIdPattern": "[cookies.spec] Cookie specs BrowserContext.cookies should find partitioned cookie",
     "platforms": ["darwin", "linux", "win32"],
-    "parameters": ["firefox"],
-    "expectations": ["SKIP"],
-    "comment": "The test relies on the default page partition key do not contain the source origin. This is not the case for Firefox."
+    "parameters": ["webDriverBiDi"],
+    "expectations": ["FAIL"],
+    "comment": "BiDi partition key requires refactor (no longer string)"
   },
   {
-    "testIdPattern": "[cookies.spec] Cookie specs Page.setCookie should set cookie with all available properties",
+    "testIdPattern": "[cookies.spec] Cookie specs BrowserContext.setCookie should set cookie with chrome partition key",
     "platforms": ["darwin", "linux", "win32"],
-    "parameters": ["firefox"],
+    "parameters": ["webDriverBiDi"],
+    "expectations": ["FAIL"],
+    "comment": "BiDi partition key requires refactor (no longer string)"
+  },
+  {
+    "testIdPattern": "[cookies.spec] Cookie specs BrowserContext.setCookie should set cookie with string partition key",
```

Comment:
string partition key should be supported in WebDriver BiDi, right? it is the sourceOrigin attribute.

Reply:
No, as I mentioned in another comment, I think it should be passed as a json with the field `sourceOrigin` set to the value and not as a string. This is the reason the comment I set there is about refactoring the BiDi cookie interface next

Reply:
I will address it before this ships, so I think this PR is ready to be landed.